### DISCUSSION TITLE
Trying to execute testsuite working with aws sdk V1.x on V2.x hitting compilation error

### DISCUSSION THREAD

**Discussion by shilpa-rangappa** [aws/aws-sdk-java-v2#3994] [Migration from v1 to v2]
### Describe the feature

Hi,

Am trying execute the testsuite which was exercising aws sdk V1.x pacakges on our platform, now trying same suite on V2.X . I changed the domain and package names, but for few packages am not getting the names in change.log. Need your help please.
com.amazonaws.ClientConfiguration;
com.amazonaws.auth.BasicAWSCredentials;
com.amazonaws.client.builder.AwsClientBuilder;
com.amazonaws.services.identitymanagement.AmazonIdentityManagement
com.amazonaws.services.identitymanagement.AmazonIdentityManagementClientBuilder
com.amazonaws.services.s3.AmazonS3
com.amazonaws.services.s3.AmazonS3ClientBuilder
com.amazonaws.services.s3.S3ClientOptions
com.amazonaws.services.s3.model.*;
com.amazonaws.services.s3.model.UploadPartRequest;
com.amazonaws.services.securitytoken.AWSSecurityTokenService
com.amazonaws.services.securitytoken.model.AssumeRoleWithSAMLResult

Thanks,
Shilpa

### Use Case

Execute the testsuite with aws packages

### Acknowledgements

- [X] I may be able to implement this feature request
- [X] This feature might incur a breaking change

### AWS Java SDK version used

2

### JDK version used

1.8.0_301

### Operating System and version

sles15

**Comment by debora-ito** [comment]
Moving this to Github Discussions, as it's a usage question.

**Comment by debora-ito** [comment]
@shilpa-rangappa please refer to our Migration documentation - https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/migration.html

It explains how the package names and client class names changed.

Let us now if you have further questions after reading the docs.
